### PR TITLE
feat: add `VideoAttributeView`

### LINKED ISSUE #534: 7.0.0 - InnertubeError: VideoAttributeView not found!
### Steps to reproduce

1.  getInfo(video_id)
2. I use age-restricted video id from Rihanna => vr1VMr28ldo

### Failure Logs

```shell
InnertubeError: VideoAttributeView not found!
This is a bug, want to help us fix it? Follow the instructions at https://github.com/LuanRT/YouTube.js/blob/main/docs/updating-the-parser.md or report it at https://github.com/LuanRT/YouTube.js/issues!
Introspected and JIT generated this class in the meantime:
class VideoAttributeView extends YTNode {
  static type = 'VideoAttributeView';

  image: {
    sources: {
      0: {
        url: string
      }
    }
  };
  image_style: string;
  title: string;
  subtitle: string;
  secondary_subtitle: {
    content: string
  };
  orientation: string;
  sizing_rule: string;
  overflow_menu_on_tap: {
    innertube_command: NavigationEndpoint
  };
  overflow_menu_a11y_label: string;
  logging_directives: {
    tracking_params: string,
    visibility: {
      types: string
    },
    enable_displaylogger_experiment: boolean
  };

  constructor(data: RawNode) {
    super();
    this.image = {
      sources: {
        0: {
          url: data.image.sources.0.url
        }
      }
    };
    this.image_style = data.imageStyle;
    this.title = data.title;
    this.subtitle = data.subtitle;
    this.secondary_subtitle = {
      content: data.secondarySubtitle.content
    };
    this.orientation = data.orientation;
    this.sizing_rule = data.sizingRule;
    this.overflow_menu_on_tap = {
      innertube_command: new NavigationEndpoint(data.overflowMenuOnTap.innertubeCommand)
    };
    this.overflow_menu_a11y_label = data.overflowMenuA11yLabel;
    this.logging_directives = {
      tracking_params: data.loggingDirectives.trackingParams,
      visibility: {
        types: data.loggingDirectives.visibility.types
      },
      enable_displaylogger_experiment: data.loggingDirectives.enableDisplayloggerExperiment
    };
  }
}

    at ERROR_HANDLER (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10488:20)
    at createRuntimeClass (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:9968:3)
    at generateRuntimeClass (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10011:19)
    at parseItem (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10754:75)
    at Object.parseArray (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10796:22)
    at new HorizontalCardList (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:1351:33)
    at parseItem (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10776:22)
    at Object.parseArray (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10796:22)
    at new StructuredDescriptionContent (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:1570:33)
    at Object.parseItem (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10776:22) {
  date: 2023-11-02T01:37:21.087Z,
  version: '7.0.0'
}
ParsingError: Type mismatch, got VideoAttributeView expected SearchRefinementCard | MacroMarkersListItem | GameCard | VideoCard.
    at ERROR_HANDLER (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10477:20)
    at parseItem (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10758:13)
    at Object.parseArray (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10796:22)
    at new HorizontalCardList (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:1351:33)
    at parseItem (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10776:22)
    at Object.parseArray (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10796:22)
    at new StructuredDescriptionContent (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:1570:33)
    at Object.parseItem (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10776:22)
    at new EngagementPanelSectionList (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:1591:35)
    at parseItem (D:\xampp\htdocs\node_modules\.pnpm\youtubei.js@7.0.0\node_modules\youtubei.js\bundle\node.cjs:10776:22) {
  info: {
    image: { sources: [Array] },
    imageStyle: 'VIDEO_ATTRIBUTE_IMAGE_STYLE_SQUARE',
    title: 'Bitch Better Have My Money',
    subtitle: 'Rihanna',
    secondarySubtitle: { content: 'Bitch Better Have My Money' },
    orientation: 'VIDEO_ATTRIBUTE_ORIENTATION_HORIZONTAL',
    sizingRule: 'VIDEO_ATTRIBUTE_SIZING_RULE_RESPONSIVE',
    overflowMenuOnTap: { innertubeCommand: [Object] },
    overflowMenuA11yLabel: 'More actions',
    loggingDirectives: {
      trackingParams: 'CCgQyMoHGAAiEwiixZDzlqSCAxVEqksFHbhfAm4=',
      visibility: [Object],
      enableDisplayloggerExperiment: true
    }
  },
  date: 2023-11-02T01:37:21.093Z,
  version: '7.0.0'
}
```

### Expected behavior

A json response with no error

### Current behavior

Got error it say's it was a bug

### Version

Default

### Anything else?

_No response_

### Checklist

- [X] I am running the latest version.
- [X] I checked the documentation and found no answer.
- [X] I have searched the existing issues and made sure this is not a duplicate.
- [X] I have provided sufficient information.

### PR DESCRIPTION
**Pull request by JellyBrick**
- Fixed an issue when getting information about this video (https://youtu.be/YsSShYaKOvk)
- resolves #534 

```typescript
InnertubeError: VideoAttributeView not found!
This is a bug, want to help us fix it? Follow the instructions at https://github.com/LuanRT/YouTube.js/blob/main/docs/updating-the-parser.md or report it at https://github.com/LuanRT/YouTube.js/issues!
Introspected and JIT generated this class in the meantime:
class VideoAttributeView extends YTNode {
  static type = 'VideoAttributeView';

  image: {
    sources: {
      0: {
        url: string
      }
    }
  };
  image_style: string;
  title: string;
  subtitle: string;
  secondary_subtitle: {
    content: string
  };
  orientation: string;
  sizing_rule: string;
  overflow_menu_on_tap: {
    innertube_command: NavigationEndpoint
  };
  overflow_menu_a11y_label: string;
  logging_directives: {
    tracking_params: string,
    visibility: {
      types: string
    },
    enable_displaylogger_experiment: boolean
  };

  constructor(data: RawNode) {
    super();
    this.image = {
      sources: {
        0: {
          url: data.image.sources.0.url
        }
      }
    };
    this.image_style = data.imageStyle;
    this.title = data.title;
    this.subtitle = data.subtitle;
    this.secondary_subtitle = {
      content: data.secondarySubtitle.content
    };
    this.orientation = data.orientation;
    this.sizing_rule = data.sizingRule;
    this.overflow_menu_on_tap = {
      innertube_command: new NavigationEndpoint(data.overflowMenuOnTap.innertubeCommand)
    };
    this.overflow_menu_a11y_label = data.overflowMenuA11yLabel;
    this.logging_directives = {
      tracking_params: data.loggingDirectives.trackingParams,
      visibility: {
        types: data.loggingDirectives.visibility.types
      },
      enable_displaylogger_experiment: data.loggingDirectives.enableDisplayloggerExperiment
    };
  }
}
```

### CODE REVIEW

File: src/parser/classes/VideoAttributeView.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  secondary_subtitle: {
+    content: string
+  };
+  orientation: string;
+  sizing_rule: string;
+  overflow_menu_on_tap: {
+    innertube_command: NavigationEndpoint
+  };
+  overflow_menu_a11y_label: string;
+  logging_directives: {
```
Comment (absidue) [inline src/parser/classes/VideoAttributeView.ts:25]: You can remove the logging_directives and it's children, generally YouTube.js avoids extracting the logging and tracking information, as nobody is really interested in it :)

File: src/parser/classes/VideoAttributeView.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { YTNode } from '../helpers.js';
+import { NavigationEndpoint } from '../nodes.js';
+
+import ContentPreviewImageView from './ContentPreviewImageView.js';
+import Parser from '../index.js';
```
Comment (absidue) [inline src/parser/classes/VideoAttributeView.ts:5]:
```suggestion
import { Parser } from '../index.js';
```

File: src/parser/classes/VideoAttributeView.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { YTNode } from '../helpers.js';
+import { NavigationEndpoint } from '../nodes.js';
```
Comment (absidue) [inline src/parser/classes/VideoAttributeView.ts:2]:
```suggestion
import NavigationEndpoint from './NavigationEndpoint.js';
```

**Review by absidue — COMMENTED** [review]
Please used named imports for the Parser instead of a default one, please see #535 for why. You only need to change the files you added, the linked pull request fixes all other ones.

**Review by LuanRT — APPROVED** [review]

**Pull request merged**